spanner_orm/admin/column.py

```diff
@@ -33,6 +34,9 @@ class ColumnSchema(schema.InformationSchema):
     is_nullable = field.Field(field.String)
     spanner_type = field.Field(field.String)
 
+    string_pattern = "^STRING\([0-9]+\)+$"
+    string_array_patter = "^ARRAY<STRING\([0-9]+\)>+$"
```

Comment:
This could just be compiled rather than compiled everytime it's used

Reply:
Right on, where do you suggest it should be placed?

Reply:
Class var (as you have) or global are both fine..

Reply:
Done 👌